app/controllers/game.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Module dependencies.
+ */
+
+import mongoose from 'mongoose';
+
+const Game = mongoose.model('Game');
+
+const gameResult = (req, res) => {
+  let gameId = req.params.id;
+  gameId = parseInt(gameId, 10);
+  const { gameStarter, gameWinner, players } = req.body;
+  if (!gameStarter || !players) {
+    return res.status(401).json({ message: 'Game have not started' });
+  }
+  const payload = {
+    gameStarter,
+    gameWinner,
+    players,
+    gameId
+  };
+  const game = new Game(payload);
+  game.save((err) => {
+    if (err) {
+      return res.status(401).json({ message: 'Error saving game' });
```

Comment:
401 isn't the right status code to be sent back when an error occurs during an operation on the server

Reply:
Done